composer.json

```diff
@@ -15,15 +15,15 @@
     ],
 
     "require": {
-        "php":                  ">=5.3.6",
+        "php":                  ">=5.5.9",
         "behat/mink":           "^1.7.1@dev",
-        "symfony/browser-kit":  "~2.3|~3.0|~4.0",
-        "symfony/dom-crawler":  "~2.3|~3.0|~4.0"
+        "symfony/browser-kit":  "^3.4|~4.0|~5.0"
     },
 
     "require-dev": {
         "mink/driver-testsuite": "dev-master",
-        "symfony/http-kernel": "~2.3|~3.0|~4.0"
+        "symfony/http-kernel": "^3.4|~4.0|~5.0",
+        "phpunit/phpunit": "<8"
```

Comment:
This seems unfortunate, but at least practical. I don't think phpunit compatibility should block releasing Symfony5 support.

Another option would be to symfony/phpunit-bridge (see https://github.com/minkphp/Mink/pull/794/files).

Reply:
The `"phpunit/phpunit": "<8"` line adds dependency on PHPUnit, but on Travis we're still using `simple-phpunit`.

In fact in other PRs, that might be merged by now we're no longer using `simple-phpunit` to get better control over installed PHPUnit version.